src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -1,2 +1,8 @@
 CHANGELOG
 =========
+
+3.3.0
+-----
+
+ * Deprecated class name support in `WorkflowRegistry::add()` as second parameter.
+   Wrap the class name in an instance of ClassInstanceSupportStrategy instead.
```

Comment:
You should probably mention the new `support_strategy` option as well (not sure if it makes sense to do it here as it's related to the config in FrameworkBundle).